README.md

```diff
@@ -11,25 +18,109 @@
           ``'"`
 ```
 
-OpenSquirrel chooses a _modular_, over a _configurable_, approach to prepare and optimize quantum circuits for heterogeneous target architectures.
+OpenSquirrel is a quantum compiler that chooses a _modular_, over a _configurable_, approach to prepare and optimize quantum circuits for heterogeneous target architectures.
 
 It has a user-friendly interface and is straightforwardly extensible with custom-made readers, compiler passes, and exporters.
 As a quantum circuit compiler, it is fully aware of the semantics of each gate and arbitrary quantum gates can be constructed manually.
 It understands the quantum programming language cQASM 3 and will support additional quantum programming languages in the future.
 It is developed in modern Python and follows best practices.
 
-
 ## Installation
 
+We encourage installing OpenSquirrel via `pip`:
+
 ```shell
 $ pip install opensquirrel
 ```
 
+To install the dependencies to run the examples on `jupyter`, install:
+
+```shell
+$ pip install opensquirrel[examples]
+```
+
+The tutorials can be found [here](https://github.com/QuTech-Delft/OpenSquirrel/tree/develop/example).
+
+## Getting started
+
+Essentially, compiling a circuit in OpenSquirrel can be seen as a 3-stage process:
+1. Defining and building the quantum circuit using either the `CircuitBuilder` or from a `cQASM` string.
+2. Executing multiple passes on the circuit, each traversing and modifying it (e.g., a decomposition).
+3. Exporting the circuit (to `cQASM` again, or to a _lowered_ language like QuantifyScheduler). 
+
+Here is an example of building a circuit using the `CircuitBuilder`:
+
+```python
+import math
+from opensquirrel.circuit_builder import CircuitBuilder
+from opensquirrel.ir import Qubit, Float
+
+# Tell the circuit builder how you want your circuit
+builder = CircuitBuilder(qubit_register_size=1)
+builder.H(Qubit(0))
+builder.Z(Qubit(0))
+builder.Y(Qubit(0))
+builder.Rx(Qubit(0), Float(math.pi / 3))
+
+# Get the circuit from the circuit builder
+circuit = builder.to_circuit()
+
+cqasm_string = (
+"""
+version 3.0
+
+qubit[1] q
+
+H q[0]
+Z q[0]
+Y q[0]
+Rx(1.0471976) q[0]
+""")
+
+qc = Circuit.from_string(cqasm_string)
```

Comment:
Can you please put this in a different code block? You can add the following text before:

``And here is an example of building a circuit from a `cQASM` string``.

Reply:
I do not know how I missed this originally hahahahaha

Reply:
Done,  nice!